.github/workflows/main.yml

```diff
@@ -331,20 +331,32 @@ jobs:
         with:
           name: ubuntu-latest-3.9
           path: /tmp/u39
+      - uses: actions/download-artifact@v2
+        with:
+          name: ubuntu-latest-3.10
+          path: /tmp/u310
       - uses: actions/download-artifact@v2
         with:
           name: macos-latest-3.8
           path: /tmp/m38
+      - uses: actions/download-artifact@v2
+        with:
+          name: macos-latest-3.10
+          path: /tmp/m310
```

Comment:
I see that now we're testing 3.8 and 3.10 on Mac (and Windows below). What about 3.9?
Or is this only temporarily for the purpose of this PR and you will remove it before merging such that we test all Python versions on Linux again + a single version on Mac and Windows?

Reply:
I decided to add 3.10 to Mac and windows permanently since it is the very latest version and were errors may occur more frequently. I am doing this for all repos. Ideally I would test all OS against all python versions but that would increase the CI time and add to the MacOS and Windows contention since there aren't many VM availables. I believe python 3.9 is close enough to python 3.8 or python 3.10 so that can be sacrificed. However if you feel python 3.9 testing is important I can add to the CI here but it will increase the time the CI runs.

Reply:
Nope all good! I mainly wanted to clarify your intent here :slightly_smiling_face: